### ISSUE TITLE
Peroneal tendon tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11153]
**Preferred term label:**
Peroneal tendon tear

**Synonyms**

**Definition (free text, please give PubMed ID)**
MRI study of the ankle showing partial or complete tear of one or both of the peroneal tendons

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ankle sprain

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001343
- label: Peroneal tendon tear
- parents: [Abnormal ankle morphology (HP:0034673)](https://hpo.jax.org/app/browse/term/HP:0034673)
- pmid: PMID:23255535
- def: Injury to a peroneal tendon, as can be ascertained by an MRI study of the ankle showing partial or complete tear of one or both of the peroneal tendons.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11153
- synonyms:
